packages/jasmine/test/BUILD.bazel

```diff
@@ -26,13 +33,30 @@ jasmine_node_test(
     shard_count = 3,
 )
 
+jasmine_node_test(
+    name = "sharding_bootstrap_test",
+    srcs = ["sharded_test.js"],
+    data = ["bootstrap.js"],
+    shard_count = 3,
+    templated_args = ["--node_options=--require=$(rlocation $(location :bootstrap.js))"],
+)
+
 jasmine_node_test(
     name = "failing_sharding_test",
     srcs = ["failing_sharded_test.js"],
     expected_exit_code = 3,
     shard_count = 2,
 )
 
+jasmine_node_test(
```

Comment:
is there a reason all these tests need to be run under the alternate bootstrap script?

Reply:
I'll add some notes as to the point. I'm being pedantic and verifying that adding a bootstrap script does not break jasmine_node_test. Regression testing mostly after the break in the angular repo being extra careful.